build-image.sh

```diff
@@ -5,12 +5,11 @@ mkdir -p repos/
 cd repos/
 
 # Clone private repositories.
-git clone git@github.com:golemfactory/hello-gwasm-runner.git
-git clone git@github.com:golemfactory/gudot.git
-git clone git@github.com:golemfactory/key_cracker_cpp.git
-git clone git@github.com:golemfactory/key_cracker_demo.git
-git clone git@github.com:golemfactory/key_cracker_gen.git
-#git clone https://github.com/golemfactory/gwasm-runner.git
+./../prepare-git-repo.sh hello-gwasm-runner
+./../prepare-git-repo.sh gudot
+./../prepare-git-repo.sh key_cracker_cpp
+./../prepare-git-repo.sh key_cracker_demo
```

Comment:
```suggestion
./../prepare-git-repo.sh key_cracker_rust
```
pls rename it to `_rust` as opossite to `_cpp`.
Whats more, there is contents of this repo duplicated within itself
```
root@d174bcaa9dda:~/key_cracker_cpp# ls -o
total 28
-rw-r--r-- 1 root   96 Oct  1 09:41 CMakeLists.txt
-rw-r--r-- 1 root   18 Oct  1 09:41 README.md
drwxr-xr-x 2 root 4096 Oct  1 10:39 bin
drwxr-xr-x 5 root 4096 Oct  1 10:39 build
drwxr-xr-x 4 root 4096 Oct  1 09:41 include
drwxr-xr-x 5 root 4096 Oct  1 09:52 key_cracker_cpp
drwxr-xr-x 2 root 4096 Oct  1 09:41 src
root@d174bcaa9dda:~/key_cracker_cpp# ls -o key_cracker_cpp/
total 16
-rw-r--r-- 1 root   96 Oct  1 09:52 CMakeLists.txt
-rw-r--r-- 1 root   18 Oct  1 09:52 README.md
drwxr-xr-x 4 root 4096 Oct  1 09:52 include
drwxr-xr-x 2 root 4096 Oct  1 09:52 src
```

And README.txt is empty....
Please at least copy wiki to README.txt
https://github.com/golemfactory/key_cracker_cpp/wiki